track_viz_2d:
  ros__parameters:
    camera_frame: CAM_FRONT
    color_dets: blue
    color_trks: red
    bbox_line_width: 5
